lib/ueberauth/strategy/fitbit/oauth.ex

```diff
@@ -5,89 +5,73 @@ defmodule Ueberauth.Strategy.Fitbit.OAuth do
   Add `client_id` and `client_secret` to your configuration:
 
   config :ueberauth, Ueberauth.Strategy.Fitbit.OAuth,
-    client_id: System.get_env("FITBIT_APP_ID"),
-    client_secret: System.get_env("FITBIT_APP_SECRET")
+    client_id: System.get_env("FITBIT_CLIENT_ID"),
+    client_secret: System.get_env("FITBIT_CLIENT_SECRET"),
   """
   use OAuth2.Strategy
+  alias OAuth2.Strategy.AuthCode
 
-  @defaults [
+  @fitbit_config [
     strategy: __MODULE__,
-    site: "https://api.fitbit.com",
+    headers: [{"Content-Type", "application/x-www-form-urlencoded"}],
+    site: "https://api.fitbit.com/",
     authorize_url: "https://www.fitbit.com/oauth2/authorize",
-    token_url: "https://api.fitbit.com/oauth2/token",
-    headers: [{"Content-Type", "application/x-www-form-urlencoded"}]
+    token_url: "https://api.fitbit.com/oauth2/token"
   ]
```

Comment:
I noticed we also have these URLs in `lib/ueberauth/strategy/fitbit.ex`. A single module or config to store this would be nice to have.

Reply:
Yeah, my bad. We just need them on the oauth module. The strategy doesn't need them